### PR TITLE
generate android pages for sitemap too

### LINKED ISSUE #16910: Include /android/ pages in the sitemap
followup for #16508 

We currently generate all urls with the app in them as `/firefox/`.  We should also include `/android/` pages too - the landing pages are different but the detail pages are essentially the same (slightly different header, with some extra warnings about compatibility)

Questions:
- Should add-ons that only support desktop Firefox have an `/android/` page?  Should purely Android compatible add-ons have a `/firefox/` page?  They do exist if navigated to but we don't link to them as they're invalid, really.
- If we have both /firefox/ and /android/ urls, should one be an "alternative" of the other or included separately?

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #16910

### CODE REVIEW

File: src/olympia/amo/sitemap.py [hunk 1]
```diff
@@ -112,34 +116,46 @@ class AMOSitemap(Sitemap):
...
-            'browse.themes.categories',
-            'browse.language-tools',  # TODO: when we add /android this is /firefox only
+            ('home', amo.FIREFOX),
+            ('home', amo.ANDROID),
+            ('pages.about', None),
+            ('pages.review_guide', None),
+            ('browse.extensions', amo.FIREFOX),
+            ('browse.extensions', amo.ANDROID),
+            ('browse.extensions.categories', amo.FIREFOX),
+            ('browse.extensions.categories', amo.ANDROID),
```
Comment (bobsilverberg) [inline src/olympia/amo/sitemap.py:131]: Did we discuss excluding categories for Android, or is this something else?

**Review by bobsilverberg — APPROVED** [review]
Looks good, sorry for the delay @eviljeff. I just have one question.

r+wc

File: src/olympia/amo/sitemap.py (hunk 1 above)
Comment (eviljeff) [reply src/olympia/amo/sitemap.py:131]: ah, this is the page that shows the _list_ of categories https://addons.mozilla.org/en-US/firefox/extensions/categories/ though I'm not sure we need that at all (or why it exists as a page).  

The addons in a particular category is handled in `CategoriesSitemap` (I thought that's what you were referring to in the issue)

Comment (eviljeff) [reply src/olympia/amo/sitemap.py:131]: I'll drop this url for Firefox too.

**Pull request merged**